src/pages/EventPage.js

```diff
@@ -37,11 +36,7 @@ function loadEvent(event) {
     if (event) {
         return (
             <div>
-                <Button variant="contained" color="primary" href="/eventform">Register!</Button>
-                <img src={event.img} alt="Event"></img>
-                <Typography>{event.ename}</Typography>
-                <Typography>{event.tagline}</Typography>
```

Comment:
Do we even need the file `EventPage` if all we do is put the event form wrapper in there?